src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -32,9 +43,13 @@ function retry () {
     shift 2
 
     while test ${i} -lt ${max} ; do
+	printf "Trying: %s\n" "$*"
         if "$@" ; then
+	    status ' SUCCESS'
             break
         fi
+	trouble '  FAILED'
+	status "Waiting ${delay} ..."
```

Comment:
Ah. So `--silent` is _too_ silent.

The main thing I'm worried about re: printing every iteration is that it would make it hard to actually look at the messages before it succeeds (because the buffer might run out or something).  If that's not an issue, I guess it's fine?